NativeApp/src/utilities/styles.js

```diff
@@ -0,0 +1,32 @@
+import { StyleSheet } from 'react-native';
+import { WHITE, INACTIVECOLOR, LIGHTGREY, UNOBLUE } from '../styles/colors';
+
+import { H2_SIZE } from '../styles/text';
+
+const styles = StyleSheet.create({
+  headerTitleStyle: {
+    color: WHITE,
+    fontSize: H2_SIZE,
+    fontFamily: 'oswaldRegular',
+  },
+  headerStyle: {
+    paddingHorizontal: 5,
+    backgroundColor: UNOBLUE,
+    borderBottomWidth: 1,
+    borderBottomColor: 'rgba(192,192,192,0.3)',
```

Comment:
Could this become a common colour or is this the only place it's used?

Reply:
This is the only place it is used.